service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerMidEventTest.java

```diff
@@ -119,6 +120,64 @@ void shouldNotPopulateChildSelectorAndLabelWhenYesIsSelected() {
 
             CaseData caseData = mapper.convertValue(callbackResponse.getData(), CaseData.class);
 
+            assertThat(callbackResponse.getData().get("children_label")).isNull();
+            assertThat(caseData.getChildSelector()).isNull();
+            assertThat(callbackResponse.getData().get("remainingChildIndex")).isNull();
+            assertThat(callbackResponse.getData().get("remainingChild")).isNull();
+            assertThat(callbackResponse.getData().get("otherFinalOrderChildren")).isNull();
```

Comment:
Do you think we could group all these `callbackResponse.getData().get(..` into `assertThat(callbackResponse.getData()).extracting(....).containsOnlyNulls();`. Would probably work for the childSelector as well